language-design.md

```diff
@@ -38,7 +55,38 @@ you know a poorly designed language? What are the symptoms?
 
 **Response**
 
-
+Language design is tricky, and it's hard to get it right. Determining which
+languages are good is important to know which ones to use --- even a DSL that is
+designed exactly for your purpose is not a good choice if it is poorly designed.
+Steele would say, and we would agree, that a good general purpose language is
+not necessarily marked by how good its syntax is or how many features it has but
+rather by its ability to grow.  No language can cover all use cases, and if
+there's a different syntax between built in types and user defined ones, overall
+awkwardness results. Steele believes that good languages are, in fact, small at
+their core and are easily extensible so users can define the features they
+need. Adam doesn't necessarily agree with this because it's nice to have
+standardized common features, although does agree that these should be part of
+standard libraries and not the language core.
+
+Domain-specific languages, on the other hand, have much less necessity to grow.
+Growing a domain-specific language would probably mean that it is being used
+outside its domain, so growth is not a priority. Being able to grow doesn't
+make it a bad language, but it's not as necessary as it is for general purpose
+languages.
+
+Robin believes that good documentation is necessary for a good language because
+you need to be able to actually use a language for it to be good. Adam
+disagrees because he believes that the goodness of a language is basically
+independent of its documentation, although he agrees that documentation is good
+for everyone involved.
```

Comment:
I would agree with Adam. A language's documentation is important for learning the language, but doesn't necessarily define the "goodness" of the language.